resources/views/pages/partials/header/preview.blade.php

```diff
@@ -27,6 +27,9 @@ class="z-10 absolute -top-4 left-8 bg-red rounded-xl font-bold text-xs text-whit
                             <x-local-time class="text-base"
                                           :date="$upcomingStream->actual_start_time ?? $upcomingStream->scheduled_start_time"/>
                         </p>
+                        <time class="text-base text-gray-dark flex ml-6" datetime="{{ $upcomingStream->actual_start_time ?? $upcomingStream->scheduled_start_time  }}">
```

Comment:
1. Please use your new "time", instead of the old one for the "stream-preview-card". I think that's enough and we don't need to show both. e.g. "Started 5min ago" or "Starts in 5min"

2. A little tricky is the calculated time. Now it compares the time with the server time, so it might be wrong from users in other timezones.

I guess best would be to create a new blade component `<x-local-time-remaining .../>` (similar to the given `<x-local-time...>`. In there we need to use JS to calculate the time difference with user's local time. (again, similar to the given time component)

I think that's the only way to show the correct remaining time, or how the stream is already live.